src/pas/plugins/ldap/properties.yaml

```diff
@@ -88,6 +88,21 @@ widgets:
         value: expr:context.users.memberOfSupport
         props:
             label: memberOf attribute supported?
+    - recursiveGroups:
+        factory: '#field:checkbox'
+        value: expr:context.users.recursiveGroups
+        props:
+            label: Support recursive/nested groups?
+            help: If your LDAP/AD supports it this will use LDAP_MATCHING_RULE_IN_CHAIN.
```

Comment:
I'm not 100% certain. I found some reference that you could use it with OpenLDAP if you add https://github.com/cbueche/openldap-passthru as discussed in this thread: https://lists.openldap.org/hyperkitty/list/openldap-technical@openldap.org/thread/MQUGRNM6XTBNHD5IXJXXS5ADIP67MI3T/#DTN3HD6KYR3BGENGFC4BWJYOWSFLD32V
 
I have no OpenLDAP to test it though. I can confirm that it works fine in AD though. It is also very effective since if saves the additional lookups that plone.app.ldap used to do.